### DISCUSSION TITLE
Biiling Error

### DISCUSSION THREAD

**Discussion by senseless69** [community/community#79233] [GitHub Education]
I Have Updated My Billing Information But Still I am unable to upload picture of my document.

**Comment by Flojan** [comment]
The problem is still there. Have you been able to find a solution?

**Comment by MaSTERmIKK** [comment]
Hey @senseless69,

I had a similar issue before. Even after updating my billing info, I couldn’t upload a document image.
What worked for me was:

Double-checking that the billing update was fully processed.

Logging out and back in again.

Trying from an incognito browser or a different device.

If that didn’t work, I contacted [GitHub Support](https://support.github.com/contact), and they found there was a backend flag on my account that needed to be cleared.

Might be worth trying the same steps. Let me know if it works!
